SECURITY_CONTACTS

```diff
@@ -0,0 +1,14 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Committee to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to abide by the
+# [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
+
+tallclair
+kubernetes/enhancements-admins
```

Comment:
Needs to be expanded too...

Reply:
I expect that this works via the OWNERS_ALIAS file too, right?

Reply:
> I expect that this works via the OWNERS_ALIAS file too, right?

Yes and no...this file has no automation involved with it and is used "manually" by the Product Security Committee to see who should be contacted in case of a security concern. I guess they can infer from OWNERS_ALIASES themselves too though. :)

Reply:
Ah okay thank you for the clarification :+1: